app/views/services/index.html.erb

```diff
@@ -1,5 +1,8 @@
 <%= render 'shared/banner' %>
 
+<div class="container pb-3 d-flex align-content-around flex-wrap justify-content-center">
```

Comment:
Maybe best to create our own new **container class**  instead of using 6 different bootstrap classes?